sql/npc_list.sql

```diff
@@ -9834,7 +9834,7 @@ INSERT INTO `npc_list` VALUES (17142582,'Count_Bifrons','Count Bifrons',0,0.000,
 INSERT INTO `npc_list` VALUES (17142583,'blank','',0,0.000,0.000,0.000,0,40,40,0,0,0,6,27,0x010003011A10192019301940195020611D700000,32,'WOTG',1);
 INSERT INTO `npc_list` VALUES (17142584,'blank','',0,0.000,0.000,0.000,0,40,40,0,0,0,6,25,0x010008021A10192019301940195020611D700000,32,'WOTG',1);
 INSERT INTO `npc_list` VALUES (17142585,'blank','',0,0.000,0.000,0.000,0,40,40,0,0,0,6,27,0x01000D021A10192019301940195020611D700000,32,'WOTG',1);
-INSERT INTO `npc_list` VALUES (17142586,'Indescript_Markings','Indescript Markings',118,-517.000,-167.000,209.000,32769,40,40,0,0,0,2,3,0x0000340000000000000000000000000000000000,0,'WOTG',1);
+INSERT INTO `npc_list` VALUES (17142586,'Indescript_Markings','Indescript Markings',118,-288.011,-56.000,-106.048,32769,50,50,0,0,0,2,3,0x0000340000000000000000000000000000000000,0,'WOTG',1);
```

Comment:
speed change also from cap?

Reply:
Yes. Everything else matched what the capture returned, except xyz pos and sepeed/speedsub